contracts/core/BaseVault.sol

```diff
@@ -296,7 +296,7 @@ contract BaseVault is ReentrancyGuard, Ownable, ERC20, Initializable {
     // This checks if there is a withdrawal
```

Comment:
The `_transfer` in `_redeem` is to transfer the shares the vault owes the user, so it's paying shares to user from vault; 

where as the `_transfer` in `initiateWithdraw` is to make sure users lock up their shares, so it's pulling shares from user to the vault. 

An easier way to think about it is that, they are 2 separated actions, calling `_redeem` as the first step in `initiateWithdraw` is not necessary, but it improve the UX: if user deposited in the middle of the round before, it can initiate a withdraw without needing to claim their shares first.

Reply:
Makes sense, thanks for explaining that.